test/test_spreadsheets.js

```diff
@@ -0,0 +1,337 @@
+                }
+            );
+            test_parse_scheduling(
+                "@chill5555 vs @doganof 15/04 at 17:00 GMT",
+                {
+                    white: "chill5555",
+                    black: "doganof",
+                    date: "2016-04-15T17:00:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@oldtom v @bramminator  04/17 @ 16:15 GMT",
+                {
+                    white: "oldtom",
+                    black: "bramminator",
+                    date: "2016-04-17T16:15:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@ctorh - @practicedave 18/4@18:00GMT",
+                {
+                    white: "ctorh",
+                    black: "practicedave",
+                    date: "2016-04-18T18:00:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@boviced v @hoxhound April 14, 16:00 GMT",
+                {
+                    white: "boviced",
+                    black: "hoxhound",
+                    date: "2016-04-14T16:00:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@pasternak v @riemannn, April 14, 20:00 GMT",
+                {
+                    white: "pasternak",
+                    black: "riemannn",
+                    date: "2016-04-14T20:00:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@angborxley v @theknug April 17th 13:30 GMT",
+                {
+                    white: "angborxley",
+                    black: "theknug",
+                    date: "2016-04-17T13:30:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@imakethenews v. @daveyjones01  April 15, 22:00 GMT, 18:00 EDT",
+                {
+                    white: "imakethenews",
+                    black: "daveyjones01",
+                    date: "2016-04-15T22:00:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@imakethenews v. @daveyjones01  April 15, 22:00 GMT, 18:00 EDT",
+                {
+                    white: "imakethenews",
+                    black: "daveyjones01",
+                    date: "2016-04-15T22:00:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "@modakshantanu v @hakonj April 13th 07:00 GMT",
+                {
+                    white: "modakshantanu",
+                    black: "hakonj",
+                    date: "2016-04-13T07:00:00+00:00"
+                }
+            );
+            test_parse_scheduling(
+                "osskjc vs Stoy Fri. 15th 8:00 GMT.",
+                {
+                    white: "osskjc",
+                    black: "Stoy",
+                    date: "2016-04-15T08:00:00+00:00"
+                }
+            );
+        });
+    });
+});
+
+
```

Comment:
there are new dependencies. I will update the README to include them. I put them in the vagrant-dependencies, but I think we should have an npm-dependencies script that we can use as well. I'll do both today.

Once the new dependencies are installed there is a "mocha" command that runs the tests for me.